02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+                "summary": "It is important to collect information about all household members, even if they are not mandatory, as it can help with troubleshooting and avoiding delays in the future."
+            },
+            "If a non-mandatory household (HH) member already receives SNAP, and the client says they share & prepare meals, what steps should be taken?": {
+                "score_sum": 0.3842315979369787,
+                "derived_questions": [
+                    "Does the client share a kitchen with his roommates?",
+                    "Do the roommates often cook and eat together in the shared kitchen?"
+                ],
+                "entire_card": "If a non-mandatory household (HH) member already receives SNAP, and the client says they share & prepare meals, what steps should be taken?\nProvide guidance and ask the following:\nDo you know if you're included on that household member's SNAP application?\nThe SNAP benefit typically includes people who live in the same household and share/prepare meals together on the same application. Is there a reason that you were not included on the same case?\nYou can either have (household member) add you to the existing SNAP case or you can apply on your own. If you apply on your own, you would only be able to use the SNAP benefit for yourself and you would likely have to provide a written statement from (household member) stating that the two SNAP benefits would be used separately.\nPlease note that the above does NOT apply to mandatory HH members (spouses residing together, etc.)",
+                "summary": "If a non-mandatory household member already receives SNAP and the client shares a kitchen with his roommates who often cook and eat together, the client may need to be included on the same SNAP application as the household member or apply on his own with a written statement from the household member stating that the SNAP benefits will be used separately."
+            },
+            "Are live-in attendants included in food stamp households?": {
+                "score_sum": 0.194722245249761,
+                "derived_questions": [
+                    "Do the roommates often cook and eat together in the shared kitchen?"
+                ],
+                "entire_card": "Are live-in attendants included in food stamp households?\nNo, roomers and live-in attendants are not included as part of the food stamp household even if they share and prepare meals with the client. Live-in attendants can apply as a separate HH for their own food stamp benefits",
+                "summary": "No, live-in attendants are not included in food stamp households, even if they share and prepare meals with the client. They can apply for their own food stamp benefits as a separate household."
```

Comment:
**Relevance to the question**: 🔴 Irrelevant
- May be mistakenly conflating "roomers" in the Guru card with "roommate" in the question

**Correctness as a summary of the Guru card**: 🟢 100% correct

**Helpfulness**: 🟠 Unhelpful